ts/IterableInboxDataModel.ts

```diff
@@ -32,11 +32,11 @@ class IterableInboxDataModel {
         return this.inboxMessages
     }
 
-    deleteItem(row: number, deleteSource: IterableInAppDeleteSource) {
+    deleteItem(id: string, deleteSource: IterableInAppDeleteSource) {
```

Comment:
This is a good fix, but I think you should've created a new function instead of modifying `deleteItem` since it's tied into the data model's internal list of messages. The data model will need to be revised heavily (which I'll handle after this PR), and changing one of its current functions to work differently is confusing and not consistent with how the other functions (`getItemCount`, `getAlItems`, etc.) works.

For this PR, we'll have to leave it as is since it's a pretty interlocked big PR.

Reply:
I will take a crack at creating a new function. We can discuss on Monday.